L_advanced_deployment.md

```diff
@@ -55,67 +56,65 @@ Now we need to update our `mix.exs` file to have all production dependencies lis
+     applications: [:phoenix, :cowboy, :logger, :postgrex, :phoenix_ecto, :phoenix_html]]
   end
 ```
+
 Doing this helps us overcome one of [exrm's common issues](https://github.com/bitwalker/exrm#common-issues) by helping exrm know about all our dependencies so that it can properly bundle them into our release. Without this, our application will probably alert us about missing modules or a failure to start a child application when we go to run our release.
 
 Even if we list all of our dependencies, our application may still fail. Typically, this happens because one of our dependencies does not properly list its own dependencies. A quick fix for this is to include the missing dependency or dependencies in our list of applications. If this happens to you, and you feel like helping the community, you can create an issue or a pull request to that project's repo.
 
-In versions of Phoenix previous to 0.8.0, we needed to take an extra step of starting our Endpoint (or router for older versions) inside our application's `start/2` function. This is no longer necessary as our Endpoint is registered as a worker which will be started by the Supervisor. We can see this in our `lib/hello_phoenix.ex` file.
-
-```elixir
-defmodule HelloPhoenix do
-  use Application
-
-  # See http://elixir-lang.org/docs/stable/elixir/Application.html
-  # for more information on OTP Applications
-  def start(_type, _args) do
-    import Supervisor.Spec, warn: false
-
-    children = [
-      # Start the endpoint when the application starts
-      worker(HelloPhoenix.Endpoint, []),
-
-      # Here you could define other workers and supervisors as children
-      # worker(HelloPhoenix.Worker, [arg1, arg2, arg3]),
-    ]
-
-    # See http://elixir-lang.org/docs/stable/elixir/Supervisor.html
-    # for other strategies and supported options
-    opts = [strategy: :one_for_one, name: HelloPhoenix.Supervisor]
-    Supervisor.start_link(children, opts)
-  end
-
-  # Tell Phoenix to update the endpoint configuration
-  # whenever the application is updated.
-  def config_change(changed, _new, removed) do
-    HelloPhoenix.Endpoint.config_change(changed, removed)
-    :ok
-  end
-end
-```
-Note: Just to be extra clear, this file is named after our application. If we had called it `MyApp`, this file would be `lib/my_app.ex`.
-
-There's one last thing we need to do before we create our release. We need to configure our Endpoint to act as a server in `config.exs`.
+We also need to configure our Endpoint to act as a server in `config.exs`.
 
 ```elixir
 # Configures the endpoint
 config :hello_phoenix, HelloPhoenix.Endpoint,
 url: [host: "localhost"],
-secret_key_base: "OOmSQ22Liduec/twplfKrEseNL2m7ivMK32ywKECyhckgQVLtBCxS3cMusKD2v8f",
+root: Path.dirname(__DIR__),
+secret_key_base: "3k5i8nuF9CbZqnkD/fs/GB6oMF9j0a9w5ldSwU5/Z28d1X3cZWTs8IiWjKr6nPF1",
 debug_errors: false,
+pubsub: [name: HelloPhoenix.PubSub,
+         adapter: Phoenix.PubSub.PG2],
 server: true
```

Comment:
From my novice perspective: Shouldn't `server: true` be set in `prod.exs`? `config.exs` applies to all environments right?